### ISSUE TITLE
YOLO retrain for UNIMIB2016 dataset has wrong predictions

### ISSUE THREAD

**Issue by jianing-sun** [allanzelener/YAD2K#127]
Hi I used retrain.py to retrain a new yolo model for UNIMIB2016 food tray dataset. But the result was really bad. In the end of the training, the loss for validation set was pretty small (around 500). Then I tested it even by using an image from the training set, but the result was really unreasonable. Does anyone know why's that? Below is the tested output image.
![20151127_114556](https://user-images.githubusercontent.com/33068301/40402980-ca356d9c-5e1c-11e8-9b1b-4e5896159db0.jpg)

Thanks a lot!

**Comment by jianing-sun** [comment]
@shadySource @allanzelener

**Comment by zhliu2018** [comment]
Did all test images show same result?
